addons/html_editor/static/src/main/link/link_plugin.js

```diff
@@ -77,21 +78,28 @@ export class LinkPlugin extends Plugin {
                 this.toggleLinkTools({ link: ev.target });
             }
         });
+        this.addDomListener(this.editable, "keydown", (ev) => {
+            if (ev.shiftKey && ev.key === "Enter") {
```

Comment:
Maybe can remove this condition because already listening to the "Enter" key down

Reply:
one condition ith or